core/storage/job/gae_models.py

```diff
@@ -81,11 +110,30 @@ def get_new_id(cls, entity_name):
 
     @property
     def is_cancelable(self):
+        """Determines if the running job can be canceled.
+
+        If a job is in queued or started state, then it can be canceled.
+
+        Returns:
+            A boolean the indicates whether the job is currently in 'queued' or 
+            'started' status.
+        """
+
         # Whether the job is currently in 'queued' or 'started' status.
         return self.status_code in [STATUS_CODE_QUEUED, STATUS_CODE_STARTED]
 
     @classmethod
     def get_recent_jobs(cls, limit, recency_msec):
+        """Get recently run jobs.
+
+        Args:
+            limit: The maximum number of jobs to return.
+            recency_msec: the time range in milliseconds
+                for resent jobs.
+        Returns:
+            A sequence of jobs.
```

Comment:
Add type information. It's probably list(...) where you'd need to figure out what the ... is.

Also, the description is vague. How is the returned value defined? If you think of a developer, they won't know what this does if the return value is just "a list of jobs"...